configs/components/_base-rubygem.rb

```diff
@@ -24,6 +24,7 @@
 # When cross-compiling, we can't use the rubygems we just built.
 # Instead we use the host gem installation and override GEM_HOME. Yay?
 pkg.environment "GEM_HOME", settings[:gem_home]
+pkg.environment "GEM_PATH", settings[:gem_home]
```

Comment:
I think a note as to why this is now necessary would be valuable. Eg, are there certain gems that were moved out of the stdlib that now require it? Was it a temp thing for the puppet and facter latest gems? Or maybe to build a gem from source? Or are you actually fixing an existing bug?